src/main/java/frc/robot/subsystems/LedSubsystem.java

```diff
@@ -0,0 +1,150 @@
+{ Color.kBlack, Color.kYellow, Color.kYellow, Color.kYellow, Color.kYellow, Color.kYellow, Color.kYellow, Color.kYellow, Color.kYellow, Color.kYellow, Color.kYellow, Color.kYellow, Color.kBlack, Color.kWhite, Color.kWhite, Color.kWhite }, 
+{ Color.kWhite, Color.kWhite, Color.kBlack, Color.kYellow, Color.kYellow, Color.kYellow, Color.kYellow, Color.kYellow, Color.kYellow, Color.kYellow, Color.kYellow, Color.kYellow, Color.kYellow, Color.kWhite, Color.kWhite, Color.kWhite }, 
+{ Color.kWhite, Color.kWhite, Color.kWhite, Color.kWhite, Color.kYellow, Color.kYellow, Color.kYellow, Color.kYellow, Color.kYellow, Color.kYellow, Color.kYellow, Color.kYellow, Color.kYellow, Color.kWhite, Color.kWhite, Color.kWhite }, 
+{ Color.kWhite, Color.kWhite, Color.kWhite, Color.kWhite, Color.kYellow, Color.kYellow, Color.kYellow, Color.kYellow, Color.kYellow, Color.kYellow, Color.kYellow, Color.kYellow, Color.kYellow, Color.kWhite, Color.kWhite, Color.kWhite }, 
+{ Color.kWhite, Color.kWhite, Color.kWhite, Color.kWhite, Color.kYellow, Color.kYellow, Color.kYellow, Color.kYellow, Color.kYellow, Color.kBlack, Color.kBlack, Color.kYellow, Color.kYellow, Color.kWhite, Color.kWhite, Color.kWhite }, 
+{ Color.kWhite, Color.kWhite, Color.kWhite, Color.kWhite, Color.kYellow, Color.kYellow, Color.kBlack, Color.kYellow, Color.kYellow, Color.kYellow, Color.kBlack, Color.kYellow, Color.kWhite, Color.kWhite, Color.kWhite, Color.kWhite }, 
+{ Color.kWhite, Color.kWhite, Color.kWhite, Color.kWhite, Color.kYellow, Color.kYellow, Color.kYellow, Color.kYellow, Color.kYellow, Color.kYellow, Color.kYellow, Color.kYellow, Color.kWhite, Color.kWhite, Color.kWhite, Color.kWhite }, 
+{ Color.kWhite, Color.kWhite, Color.kWhite, Color.kWhite, Color.kWhite, Color.kYellow, Color.kYellow, Color.kBlack, Color.kBlack, Color.kYellow, Color.kYellow, Color.kWhite, Color.kWhite, Color.kWhite, Color.kWhite, Color.kWhite }, 
+{ Color.kWhite, Color.kWhite, Color.kWhite, Color.kWhite, Color.kWhite, Color.kYellow, Color.kYellow, Color.kBlack, Color.kBlack, Color.kYellow, Color.kYellow, Color.kWhite, Color.kWhite, Color.kWhite, Color.kWhite, Color.kWhite }, 
+{ Color.kWhite, Color.kWhite, Color.kWhite, Color.kWhite, Color.kYellow, Color.kYellow, Color.kYellow, Color.kYellow, Color.kYellow, Color.kYellow, Color.kYellow, Color.kWhite, Color.kWhite, Color.kWhite, Color.kWhite, Color.kWhite }, 
+{ Color.kWhite, Color.kWhite, Color.kWhite, Color.kWhite, Color.kBlack, Color.kBlack, Color.kBlack, Color.kYellow, Color.kYellow, Color.kYellow, Color.kBlack, Color.kWhite, Color.kWhite, Color.kWhite, Color.kWhite, Color.kWhite }
+};
+  private LedMode ledMode = LedMode.CONE;
+  private final BlinkinDriver blinkin;
+  private BlinkinLedMode mode = BlinkinLedMode.SOLID_YELLOW;
+  /** Configure AddressableLED */
+  public LedSubsystem() {
```

Comment:
Mixing Blinkin and AddressableLED doesn't really make sense, especially since the PWM ports currently conflict (you're using port 1 for both). 

I'd keep two subsystems, one for Blinkin, and one for AddressableLED.  Then only instantiate the one you want to use/test in RobotContainer. You can then setup different methods for different LED types/arrangements. 

Alternately, you can include a constructor parameter that selects whether to use Blinkin or AddressableLED in the subsystem. For example:
```
public LedSubsystem(boolean blinkinNotAddressableLed) {
  if (blinkinNotAddressableLed) {
     blinkin = new BlinkinDriver(1);
  } else {
    matrixLeds = new AddressableLED(1);
    // other AddressableLED setup
  }
}
```
I'm not sure what the other AddressableLED is meant for, but you could add another parameter for that as well.